WcaOnRails/db/seeds/development/users.seeds.rb

```diff
@@ -31,20 +31,15 @@ def self.random_user
 
 # Create members and leaders for every WCA team
 UsersController.WCA_TEAMS.each do |team|
-  leader_team = :"#{team}_leader"
-  leader = User.new(random_user.merge(
-    team => true,
-    leader_team => true,
-    name: team.to_s,
-  ))
-  leader.skip_confirmation!
-  leader.save!
+  team_id = Team.find_by_friendly_id(team).id
```

Comment:
Also, I think this seed file now depends on running *after* the new `teams.seeds.rb` file, so it would probably be best to wrap this while file in a call to `after :teams` (see https://github.com/james2m/seedbank#usage).